### DISCUSSION TITLE
Nuclei template is not updated in time

### DISCUSSION THREAD

**Discussion by Bywalks** [projectdiscovery/nuclei-templates#3626] [Q&A] [answered]
Hi ,
I find nuclei template v8.8.4 is updated ,but when i update nuclei temlpate by nuclei , i can update ,why exist this situation?
Nuclei template is updated half hour.

**Comment by ehsandeep** [comment, marked as answer]
@Bywalks the version check API is behind cloudflare API to serve cached version of data and it take sometime to sync from GitHub release.
